dbms/src/Functions/FunctionsRound.h

```diff
@@ -974,8 +974,8 @@ struct TiDBFloatingRound
 
     static OutputType eval(InputType input, FracType frac)
     {
-        // modified from <https://github.com/pingcap/tidb/blob/26237b35f857c2388eab46f9ee3b351687143681/types/helper.go#L33-L48>.
-
+        // modified from https://github.com/pingcap/tidb/blob/26237b35f857c2388eab46f9ee3b351687143681/types/helper.go#L33-L48 and
+        // https://github.com/pingcap/tidb/blob/26237b35f857c2388eab46f9ee3b351687143681/types/helper.go#L50-L61.
```

Comment:
Seems that `ConstPowOf10` can be used to optimize the code in Line 896 `base = std::pow(10.0, frac)`

Reply:
It needs more effects to make sure correctness and performance improvement. Maybe we should to it in another pr

Reply:
create a new issue to record: https://github.com/pingcap/tiflash/issues/9860